README.md

```diff
@@ -157,7 +158,7 @@ thread, instead will run Async, so if the function createImageBitmap fail, it
 will not stop the rendering process at all and may cause some bitmap loss or
 simply will not draw anything in canvas, mostly on low end devices.
 
-**WARNING: Experimental, not stable and not working in Safari**
+**WARNING: Experimental, not stable and not working in some browsers**
```

Comment:
No. As written before and in the commit message it **does work** in current WebKit *(Safari uses WebKit; on MacOS/iOS the Apple build-flavour, otherwise GTK iirc)*. The only browsers where it doesn't is those using an incomplete NIH-engine or *historic* versions of WebKit, Blink or Gecko *(or Servo which currently only has an incomplete implementation from what I can gather)*.